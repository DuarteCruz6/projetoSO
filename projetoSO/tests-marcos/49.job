WRITE [(q3,77)(h4,40)(d0,29)]
WRITE [(m4,48)(w5,85)(p1,8)(p5,82)(n3,89)]
WRITE [(w1,23)(x4,41)]
DELETE [w5]
WRITE [(n7,88)]
WRITE [(a6,29)(v9,87)(a7,57)(a2,65)]
WAIT 3000
WRITE [(j2,55)(v6,72)(m7,13)(x7,66)]
WRITE [(s3,33)]
WRITE [(y0,28)]
READ [v6]
READ [d0]
WRITE [(m8,40)(l5,99)]
READ [m7]
WRITE [(k8,75)]
DELETE [v6,s3,n7,q3,d0]
DELETE [m4,w1,a2,x4]
READ [k8]
BACKUP
DELETE [m8,v9]
WRITE [(u9,89)(u2,56)]
READ [a6]
WAIT 1000
WRITE [(g2,48)]
SHOW
DELETE [a6,p1,u9,p5,a7]
DELETE [x7,y0,h4]
WRITE [(b2,82)(b4,76)]
WRITE [(i0,16)]
READ [g2]
WRITE [(p7,27)]
WRITE [(z3,44)(m4,22)(e6,43)(b1,44)]
WRITE [(p5,46)]
READ [p5]
WRITE [(b7,3)(o5,82)(f2,70)]
SHOW
READ [l5]
WRITE [(o8,66)(j6,17)(n4,66)(b3,45)]
WRITE [(d5,43)(s2,25)(r2,88)(v1,78)]
READ [p5]
