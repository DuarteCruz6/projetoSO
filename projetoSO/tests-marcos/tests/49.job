BACKUP
SHOW
WRITE [(c0,24)(b3,43)]
READ [b3]
READ [b3]
READ [b3]
DELETE [c0,b3]
WRITE [(c3,40)(t5,87)(r9,25)]
WRITE [(c9,76)(p1,87)(p8,61)(e6,47)]
WRITE [(h8,45)(l4,65)(z9,31)]
BACKUP
WRITE [(u3,32)]
READ [u3]
DELETE [l4,t5,c3]
WRITE [(h4,97)(i7,43)(b7,35)]
WRITE [(w4,47)]
WRITE [(t3,62)(h7,57)(r4,63)(l8,15)]
WAIT 2000
READ [h7]
WRITE [(g2,87)(e4,21)(q1,93)(m9,18)]
WRITE [(r1,45)]
READ [z9]
WRITE [(x2,64)(d8,42)(n8,85)(g4,53)(a4,8)]
SHOW
WRITE [(b1,82)(h0,61)(y4,3)(h5,33)(p4,26)]
READ [p8]
READ [r9]
WRITE [(o6,18)(a5,98)]
WRITE [(v9,36)(i6,97)(s8,55)(z8,9)]
WRITE [(u9,96)(v5,93)(w0,59)]
READ [z9]
WRITE [(q9,15)(y5,25)(i8,26)(k8,60)]
READ [y5]
WRITE [(s7,22)(z6,28)]
DELETE [r9,n8]
WRITE [(o9,58)(r3,13)]
WRITE [(r8,16)(n0,68)]
WRITE [(s3,84)(i0,67)(u7,17)(w1,22)(l5,76)]
WRITE [(j2,36)(g8,26)]
READ [h0]
DELETE [h7,h8,x2]
BACKUP
WAIT 3000
WRITE [(p9,9)(q6,10)]
READ [b7]
DELETE [g2,v5,k8]
WRITE [(z7,23)(l0,62)(i9,1)(c8,11)(l4,29)]
READ [b7]
WRITE [(n2,53)(x4,83)(r2,31)(p0,46)(y2,22)]
DELETE [l8,l5]
READ [c8]
READ [w0]
READ [p9]
WRITE [(b2,18)(k6,65)(m0,20)(c6,75)]
WRITE [(x7,62)(z3,55)(i2,62)(s2,1)]
WRITE [(o7,48)(h7,45)(h2,55)(n1,90)(u4,1)]
READ [u7]
DELETE [x4]
WRITE [(j8,76)(z4,93)(w6,85)(h6,66)]
WRITE [(y6,1)(s1,19)(u6,23)]
WRITE [(v1,92)(q3,69)(o8,40)(u1,15)]
WAIT 2000
WAIT 1000
WRITE [(y9,42)]
DELETE [p4,w0,z6]
WRITE [(s6,39)(t4,38)(l6,11)(f1,65)]
READ [o9]
DELETE [u3]
DELETE [i0,k6,r4]
WRITE [(q5,54)(o5,28)(g6,70)(t8,34)]
WRITE [(d4,53)]
WRITE [(m5,23)(b9,87)(d7,27)(a8,89)(d0,31)]
READ [b9]
WRITE [(e8,45)(k5,34)(r9,87)(p7,17)(q8,54)]
