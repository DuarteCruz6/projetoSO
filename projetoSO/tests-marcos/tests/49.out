[(b3,43)]
[(b3,43)]
[(b3,43)]
[(u3,32)]
[(h7,57)]
[(z9,31)]
(a4, 8)
(b7, 35)
(c9, 76)
(d8, 42)
(e4, 21)
(e6, 47)
(g4, 53)
(g2, 87)
(h7, 57)
(h4, 97)
(h8, 45)
(i7, 43)
(l8, 15)
(m9, 18)
(n8, 85)
(p8, 61)
(p1, 87)
(q1, 93)
(r1, 45)
(r4, 63)
(r9, 25)
(t3, 62)
(u3, 32)
(w4, 47)
(x2, 64)
(z9, 31)
[(p8,61)]
[(r9,25)]
[(z9,31)]
[(y5,25)]
[(h0,61)]
[(b7,35)]
[(b7,35)]
[(c8,11)]
[(w0,59)]
[(p9,9)]
[(u7,17)]
[(o9,58)]
[(b9,87)]
